src/controllers/users.controller.js

```diff
@@ -189,6 +187,57 @@ class UserController {
     );
   }
 
+  /**
+   * @name updateUserInfo
+   * @description Updates user profile to complete registration
+   * @param {object} req The request object
+   * @param {object} res The response object
+   * @returns {object} The API response
+   */
+  async updateUserInfo(req, res) {
+    const userData = { ...req.body };
+    const { user } = res.locals;
+
+    userData.password = hash.generateSync(userData.password);
+    const data = await UserServices.updateUserInfoByEmail({ ...userData }, user.email);
+
+    if (data === 'own_manage') {
+      return Responses.handleError(409, 'You cannot manage your self', res);
+    }
+    return Responses.handleSuccess(201, 'Updated successfully', res, data);
+  }
+
+  /**
+   * @name getUserProfile
+   * @description gets user profile to complete registration
+   * @param {object} req The request object
+   * @param {object} res The response object
+   * @returns {object} The API response
+   */
+  async getUserProfile(req, res) {
+    let { userId } = req.params;
+
+    userId = Number(userId);
+
+    if (typeof userId === 'number' && userId > 0 && userId >>> 0 === userId) {
+      const userData = await UserServices.getUserById(userId);
+
+      if (userData) {
+        const user = userData.dataValues;
+
+        user.lineManager = user.LineManager ? `${user.LineManager.firstName} ${user.LineManager.lastName}` : 'none';
```

Comment:
seems you assigned this value here and just deleted it in the next line of code? not sure I understand why you need to do this.

Reply:
No I added the object `lineManager` and removed the id field `lineManagerId`.